CONTRIBUTING.md

```diff
@@ -32,11 +32,10 @@ refresh the patches we carry in this repository.
    are extending an existing patch).
 
 1. Produce an updated patch and copy it back under `third_party/llvm-project/`.
-   Using `--no-prefix` is required so `tools/setup_llvm_clone.sh` can apply the
-   patch without stripping path components:
+   To generate the patch, run this:
 
    ```sh
-   git show HEAD --no-prefix > /path/to/zkir/repo/third_party/llvm-project/<descriptive_name>.patch
+   git show HEAD > /path/to/zkir/repo/third_party/llvm-project/<descriptive_name>.patch
```

Comment:
I found that it should contain `--pretty=""` to remove the header which has metadata (author, commit name, etc).
Maybe it is different for other git versions, but for keeping consistency, how about adding this?

Reply:
I think this is out of scope of this PR, so I'll make another PR for this!